AEPCore/Tests/MobileCoreTests.swift

```diff
@@ -224,19 +224,18 @@ class MobileCoreTests: XCTestCase {
         """
         let expectedDict = jsonStrToDict(jsonStr: expected)
 
-        // test
-        MobileCore.registerExtensions([MockExtension.self, MockExtensionTwo.self], {
-            EventHub.shared.getExtensionContainer(MockExtension.self)?.registerListener(type: EventType.hub, source: EventSource.sharedState) { event in
```

Comment:
By doing it this way, there is a slight chance MockExtension.self might miss the shared state event.